docs/application/dotnet/get-started/application-filtering.md

```diff
@@ -14,7 +14,7 @@ Some features can be selectively supported by the Tizen device manufacturer. To
 
     Be careful when defining the feature list for feature-based filtering. The feature list can dramatically reduce your chances of getting the application downloaded by reducing the number of devices which can support the application.
 
-If the `tizen-manifest.xml` file of the application package includes a feature list, Tizen Store compares the capabilities of the device with the required feature conditions of the application. The store only lists the applications whose conditions match the capabilities of the device, and thus prevents incompatible applications from being installed.
+If the `tizen-manifest.xml` file of the application package includes a feature list, Tizen Store compares the capabilities of the device with the required feature conditions of the application. The store only lists the applications whose conditions match the capabilities of the device and thus prevents incompatible applications from being installed.
```

Comment:
Hello, @asm-foysal 
I have one request related with "Tizen Store"

"Tizen Store" is EOS, but the mechanism of the Tizen Store, such as filtering out the apps by feature is still valid.
For example the filtering out is working on TV Seller Office.

So, would you please change the "Tizen Store" strings in application-filtering.md files for native/web/dotnet?

For the first appearance, "Tizen Store" can be changed to "Official site for Tizen application".
After the first appearance, I think you can choose the word to describe the notion.
As an example, you don't need to change the **store** of the line 17,
as it is describing store itself, not **Tizen Store**.

```
The store only lists the applications...
```

Reply:
Hi @mijin85cho !
Thank you for the suggestion. 
We'll look into it and change the strings for "Tizen Store".
We'll even look into files other than the `application-filtering.md` file where they have used the string "Tizen store" and see if it requires changing. 
Our plan is to merge this PR and open up another PR to address these changes, what do you think?